src/page-content/navigation/navigation-announcement.tsx

```diff
@@ -3,9 +3,9 @@ import React from 'react'
 import { PropTypes as AnnouncementProps } from '../../components/layouts/navigation/announcement'
 
 const jobs: AnnouncementProps = {
-  longText: 'Ory is hiring - shape the future of identity with us!',
-  shortText: 'Ory is hiring!',
-  link: '/jobs?mtm_campaign=job-banner'
+  longText: 'Get Ory Cloud Start Up subscription free for a year using code FIRST900',
+  shortText: 'Ory Cloud Start Up Plan!',
```

Comment:
```suggestion
  shortText: 'Ory Cloud free for a year!',
```

sounds more exciting ;-)